### DISCUSSION TITLE
Contenido tests parte 2

### DISCUSSION THREAD

**Discussion by Maquin01** [IIC2133-PUC/2024-2#51] [Tarea 0] [answered]
Hola!!
En que se diferencian los test "hard" de los "easy" y "medium" de la parte 2? (además de la cantidad de datos)
Los dos medium y easy me corren bien pero el hard me avisa de diferencias de output (y como son 2 millones de lineas no estoy logrando ver en que falla mi codigo) 
Podrian ayudarme con esto o orientarme en algún sentido?
Gracias :)

**Comment by 4n1t4** [comment, marked as answer]
Principalmente se diferencian en la frecuecnia de ciertos comandos. A medida que aumenta la la dificultad de van agregando nuevos eventos que en los otros test no, de forma específica los hard , a diferencia de los otros, considera el evento GIVE_PUFFLE entre los potenciales inputs. Para ver la diferencia específica utiliza el comando -diff con los archivos, te ayuda de mejor forma a visualizar los errores de output.

**Reply by Maquin01** [reply]
Era efectivamente la funcion give_puffle la con problemas. Solucionado :)
Muchas gracias
